frontend/src/lib/components/DateFilter/DateFilterCustom.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect, useRef, useState } from 'react'
+import moment from 'moment'
+import { Button, DatePicker } from 'antd'
+
+export function DateFilterCustom(props: {
```

Comment:
It may just be the naming, but the purpose of `Custom` vs. `Component` is not immediately clear to me.

Reply:
It's indeed not clear!

I moved the custom `/insights` date filter logic inside `scenes/insights`! 
And renamed `DateFilterComponent` -> `DateFilter` and `DateFilterCustom` -> `DateFilterRange`!